src/components/CompareResults/TableContent.tsx

```diff
@@ -16,17 +16,14 @@ type ResultsForRevision = {
 };
 
 function processResults(results: CompareResultsItem[]) {
-  const processedResults: Map<string, CompareResultsItem[]> = new Map<
-    string,
-    CompareResultsItem[]
-  >();
+  const processedResults: Map<string, CompareResultsItem[]> = new Map();
   results.forEach((result) => {
     const { new_rev: newRevision, header_name: header } = result;
     const rowIdentifier = header.concat(' ', newRevision);
-    if (processedResults.has(rowIdentifier)) {
-      (processedResults.get(rowIdentifier) as CompareResultsItem[]).push(
-        result,
-      );
+    const resultsForRow = processedResults.get(rowIdentifier);
+
+    if (resultsForRow) {
+      resultsForRow.push(result);
```

Comment:
This also has the nice side effect that we don't need to cast the result of `get`, because we test for it directly.